Source/Core/GeometryPipeline.js

```diff
@@ -1974,6 +1978,11 @@ define([
 
     function splitLongitudeTriangles(instance) {
         var geometry = instance.geometry;
+
+        if (!define(geometry.indices)) {
```

Comment:
I don't think checking `geometry.indices` is the right answer.  Indices is optional.  You should check for `geometry.attributes.position` instead.  Do that throughout.